kolla-ansible/trilio-datamover/tvault-contego.conf.s3.sample

```diff
@@ -13,6 +13,10 @@ vault_s3_access_key_id =
 vault_s3_secret_access_key = 
 vault_s3_region_name = 
 vault_s3_bucket = 
+
+##'vault_s3_endpoint_url' parameter is valid only for non amazon S3 backup targets 
+vault_s3_endpoint_url = 
+
```

Comment:
Dont we need to these options, in conf file : 
vault_s3_signature_version = 
vault_s3_ssl = 

We have these optins in UI & Shell scripts .

Reply:
Added now.